crypto/err/err.c

```diff
@@ -293,8 +293,6 @@ static void ERR_STATE_free(ERR_STATE *s)
 
 DEFINE_RUN_ONCE_STATIC(do_err_strings_init)
 {
-    if (!OPENSSL_init_crypto(0, NULL))
-        return 0;
```

Comment:
Why is this not needed?

Reply:
All we do in this function is to create a lock and a hash table.  There's zero need to initialize the rest of libcrypto here, specifically.  Also, lockup.